pages/index.tsx

```diff
@@ -1,9 +1,9 @@
-import Head from 'next/head';
-import Image from 'next/image';
+// import Head from 'next/head';
+// import Image from 'next/image';
 import Hero from '../components/home/hero';
 import About from '../components/home/about';
-import Team from '../components/home/team';
-import Companies from '../components/home/companies';
+// import Team from '../components/home/team';
+// import Companies from '../components/home/companies';
```

Comment:
```suggestion
```

remove if not needed

Reply:
Eh, they aren't harming anything and they link to components that do exist, but were never finished. I'd rather leave them alone as a reminder of where to add them back someday.

Reply:
sounds good